docs/comparisons/dask.rst

```diff
@@ -59,7 +59,7 @@ Dask DataFrame
 Dask DataFrame uses row-based partitioning, similar to Spark. This can be seen in their
 `documentation`_. They also have a custom index object for indexing into the object,
 which is not pandas compatible. Dask DataFrame seems to treat operations on the
-DataFrame as MapReduce operations, which is a good paradigm for the subset of the pandas
+DataFrame as TreeReduce operations, which is a good paradigm for the subset of the pandas
```

Comment:
Why was `MapReduce` renamed to `TreeReduce`?

Reply:
`TreeReduce` is a more descriptive way to describe the pattern expected. `MapReduce` has certain connotations related to defining number of mappers/reducers, etc. I think we want to avoid any concerns there.

Reply:
I see, that makes sense, thanks!